### PR TITLE
 feat: new string validations

### PR DESCRIPTION
**Pull request by gcandal**

### CODE REVIEW

**Comment by gcandal** [comment]
@joamag PTAL

File: test/util/validation.js [hunk 1]
```diff
@@ -124,6 +124,54 @@ describe("#isIn()", function() {
...
+        assert.strictEqual(result, true);
+
+        result = yonius.isLower()("hello.hello");
+        assert.strictEqual(result, true);
+
+        result = yonius.isLower()(null);
+        assert.strictEqual(result, true);
+
+        assert.throws(
+            () => yonius.isLower()("ILLEGAL!"),
```
Comment (joamag) [inline test/util/validation.js:165]: please keep using the typical hello and hello world string "HELLO" example

File: test/util/validation.js [hunk 2]
```diff
@@ -213,3 +261,72 @@ describe("#isRegex()", function() {
...
 });
+
+describe("#stringGt()", function() {
+    it("should verify basic greater than operations", () => {
+        let result;
+
+        result = yonius.stringGt(2)("xxx");
+        assert.strictEqual(result, true);
+
+        result = yonius.stringGt(4)("xxxxx");
```
Comment (joamag) [inline test/util/validation.js]: keep using the hello world theme

File: test/util/validation.js [hunk 3]
```diff
@@ -213,3 +261,72 @@ describe("#isRegex()", function() {
...
+
+describe("#stringLt()", function() {
+    it("should verify basic greater than operations", () => {
+        let result;
+
+        result = yonius.stringLt(2)("x");
+        assert.strictEqual(result, true);
+
+        result = yonius.stringLt(4)("xxx");
+        assert.strictEqual(result, true);
```
Comment (joamag) [inline test/util/validation.js:297]: same

**Review by joamag — CHANGES_REQUESTED** [review]

File: test/util/validation.js (hunk 1 above)
Comment (gcandal) [reply test/util/validation.js:165]: Taken from https://github.com/hivesolutions/yonius/blob/35e9dcb87477180b2d3061f19d5075f068b57cca/test/util/validation.js#L141

**Comment by gcandal** [comment]
@joamag

**Review by joamag — APPROVED** [review]

**Pull request merged**